config/default/rbac/rbac_role.yaml

```diff
@@ -0,0 +1,177 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
I see that config/manager/manager.yaml → config/default/manager/manager.yaml was `File renamed without changes.` but for `config/default/rbac/rbac_role.yaml` and `config/default/rbac/rbac_role_binding.yaml` they are just newly created, no deletes for the old ones. In fact they still exist in https://github.com/kudobuilder/kudo/tree/6254e79ad747dcc0257c2587d4fbd42c7541c55c/config/rbac 

Shouldn't we move them under default entirely to avoid redundancy, or is this intended to maybe support earlier Kustomize versions (e.g. v1)? If so then we should probably also have the `manager` folder/files copied at its old place.

Reply:
that's odd.. they were moved and modified... correcting now.